src/bindings/glib/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ else()
 	target_compile_options (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_CFLAGS_OTHER})
 
 	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC elektra-core elektra-kdb)
-	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LDFLAGS})
+	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LIBRARIES})
```

Comment:
`GOBJECT_LIBRARIES` will be missing important linker flags, so linking might break. I've spent quite some time (read: hours) investigating into the different variables pkg-config is returning. `_LIBRARIES` will be insufficient.